/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loggers/wandb.py:396: UserWarning: There is a wandb run already in progress and newly created instances of `WandbLogger` will reuse this run. If this is not desired, call `wandb.finish()` before instantiating `WandbLogger`.
  rank_zero_warn(
GPU available: False, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
Sanity Checking DataLoader 0:   0% 0/2 [00:00<?, ?it/s]
  | Name         | Type      | Params
-------------------------------------------
0 | bert         | BertModel | 89.1 M
1 | hidden_layer | Linear    | 49.2 K
2 | layer        | Linear    | 65
3 | criterion    | BCELoss   | 0
-------------------------------------------
7.1 M     Trainable params
82.0 M    Non-trainable params
89.2 M    Total params


Epoch 0:   0% 0/4 [00:00<?, ?it/s]
/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/fit_loop.py:280: PossibleUserWarning: The number of training batches (4) is smaller than the logging interval Trainer(log_every_n_steps=50). Set a lower value for log_every_n_steps if you want to see logs for the training epoch.



Epoch 0: 100% 4/4 [03:21<00:00, 50.45s/it, v_num=awq5, train/loss=0.553]

        [0.5672],
        [0.5410],
        [0.5315],
        [0.5412],
        [0.5513],
        [0.5248],
        [0.5623],
        [0.5569],
        [0.5565],
        [0.5248],
        [0.5445],
        [0.5344],
        [0.5535],
        [0.5425],
        [0.5477]], grad_fn=<SigmoidBackward0>), tensor([[0.5602],
        [0.5628],
        [0.5646],
        [0.5495],
        [0.5544],
        [0.5374],
        [0.5367],
        [0.5743],
        [0.5285],
        [0.5303],
        [0.5486],
        [0.5538],
        [0.5612],
        [0.5579],
        [0.5671],
        [0.5735]], grad_fn=<SigmoidBackward0>), tensor([[0.5719],
        [0.5601],
        [0.5620],
        [0.5422],
        [0.5517],
        [0.5787],
        [0.5569],
        [0.5815],
        [0.5542],
        [0.5697],
        [0.5620],
        [0.5713],
        [0.5739],
        [0.5682],
        [0.5554],
        [0.5650]], grad_fn=<SigmoidBackward0>), tensor([[0.5749],
        [0.5870],
        [0.5690],
        [0.5856],
        [0.5784],
        [0.5616],
        [0.5827],
        [0.5706],
        [0.5748],
        [0.5467],
        [0.5808],
        [0.5888]], grad_fn=<SigmoidBackward0>)]
[tensor([0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]), tensor([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]), tensor([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 1, 1, 1, 1, 1]), tensor([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1])]
Error executing job with overrides: []
Traceback (most recent call last):
  File "/content/drive/MyDrive/murata_labo_exp/murata_labo_exp_src/exp3/main.py", line 397, in main
    trainer.fit(model, data_module)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 529, in fit
    call._call_and_handle_interrupt(
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/call.py", line 42, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 568, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 973, in _run
    results = self._run_stage()
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 1016, in _run_stage
    self.fit_loop.run()
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/fit_loop.py", line 202, in run
    self.on_advance_end()
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/fit_loop.py", line 368, in on_advance_end
    call._call_lightning_module_hook(trainer, "on_train_epoch_end")
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/call.py", line 144, in _call_lightning_module_hook
    output = fn(*args, **kwargs)
  File "/content/drive/MyDrive/murata_labo_exp/murata_labo_exp_src/exp3/main.py", line 220, in on_train_epoch_end
    epoch_preds = torch.stack(self.train_step_outputs_preds)
RuntimeError: stack expects each tensor to be equal size, but got [16, 1] at entry 0 and [12, 1] at entry 3
Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.